docs/platforms/javascript/guides/react/features/component-tracking.mdx

```diff
@@ -6,6 +6,18 @@ description: "Learn how Sentry's React SDK allows you to monitor the rendering p
 
 Sentry helps you track your React components and unlock additional insights in your application. You can set it up to track React component names as well as monitor the performance of your React components with component tracking.
 
+So instead of looking at selectors like this:
+
+```html
+div.row > div.col-sm-8.col-md-12.col-lg-8.col-xl-9 > div#main > h3#ignore-exceptions > a[aria-label="common options permalink"]
+```
+
+You can also see exactly which React component was used, like:
+
+```html
+CommonOptions
```

Comment:
In Breadcrumbs, we still want to show the entire component "tree"

Reply:
What would that look like? Could you add an example?

Reply:
oh this is good now! 
before the selector example had the entire html tree so it would make more sense if using React component example still kept the tree, but since we're only showing the selector for the before, it matches up!

Reply:
Makes sense! thanks